lisa/features/serial_console.py

```diff
@@ -183,6 +187,24 @@ def check_initramfs(
                 f"{initramfs_logs} {filesystem_exception_logs}"
             )
 
+    def check_ata_exception(
```

Comment:
Why will it be listed separately? Add code comments as well.

Reply:
When the image is ubuntu 22.04 and vm size is A1_V2, I found there are no 'initramfs' in the console log when having the same issue. So I wanted to add a new check. 

After consideration, adding ata exception check is not a good way because this is not a common check. It should not appear in the case. I have reverted it. Let me post another PR.